conda/dev.yaml

```diff
@@ -1,14 +1,14 @@
 name: sdx
 channels:
-  - nodefaults
   - conda-forge
+  - defaults
```

Comment:
actually it is not super good to mix defaults and conda-forge .. they use different version of compilers and it breaks the entire environment.
not sure if they are now synchronized .. otherwise it will still be a problem.
but for now I will merge this .. and then I will take a look into that later